regex-assembly/933210.ra

```diff
@@ -0,0 +1,31 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+##! Helpers
+##!> define single_or_double_quotes ['"]
+##!> define string_in_round_brackets \(.+\)
+
+##!$ ;
+
+##!> assemble
+    ##! example payload: (system)(ls);
```

Comment:
Fix indentation by running `crs-toolchain regex format -a`. I've just opened a PR to fix format linting n GitHub.

Reply:
done